--- termit-2.0.0/src/CMakeLists.txt~	2008-10-14 19:06:40.000000000 +0200
+++ termit-2.0.0/src/CMakeLists.txt	2008-10-14 19:11:07.000000000 +0200
@@ -10,7 +10,13 @@
   message(FATAL_ERROR "vte library not found")
 ENDIF(NOT VTE_FOUND)
 
-INCLUDE(FindLua51)
+# FindLua51 doesn't produce `-I/usr/include/lua51' so is probably broken 
+#INCLUDE(FindLua51)
+#IF(NOT LUA51_FOUND)
+#  message(FATAL_ERROR "lua 5.1 not found")
+#ENDIF(NOT LUA51_FOUND)
+
+pkg_search_module(LUA51 REQUIRED lua51>=5.1)
 IF(NOT LUA51_FOUND)
   message(FATAL_ERROR "lua 5.1 not found")
 ENDIF(NOT LUA51_FOUND)
@@ -18,7 +24,7 @@
 INCLUDE(Sources.cmake)
 
 SET(TERMIT_CFLAGS "")
-foreach(cflag ${VTE_CFLAGS} ${GTK_CFLAGS})
+foreach(cflag ${VTE_CFLAGS} ${GTK_CFLAGS} ${LUA51_CFLAGS})
   set(TERMIT_CFLAGS "${TERMIT_CFLAGS} ${cflag}")
 endforeach(cflag)
 
@@ -26,7 +32,7 @@
   SET(TERMIT_CFLAGS "${TERMIT_CFLAGS} -g -D DEBUG")
 ENDIF(DEBUG)
 
-set(TERMIT_LDFLAGS ${VTE_LDFLAGS} ${GTK_LDFLAGS})
+set(TERMIT_LDFLAGS ${VTE_LDFLAGS} ${GTK_LDFLAGS} ${LUA51_LDFLAGS})
 
 SET_SOURCE_FILES_PROPERTIES(${TERMIT_SRCS} PROPERTIES 
   COMPILE_FLAGS "${TERMIT_CFLAGS}")
@@ -38,7 +44,7 @@
 #set_target_properties(${TARGET} PROPERTIES LINK_FLAGS ${TERMIT_LDFLAGS})
 
 SET(TERMIT_LDFLAGS "")
-FOREACH(ldflag ${VTE_LDFLAGS} ${GTK_LDFLAGS})
+FOREACH(ldflag ${VTE_LDFLAGS} ${GTK_LDFLAGS} ${LUA51_LDFLAGS})
   SET(TERMIT_LDFLAGS "${TERMIT_LDFLAGS} ${ldflag}")
 ENDFOREACH(ldflag)
 set_target_properties(${TARGET} PROPERTIES LINK_FLAGS "${TERMIT_LDFLAGS}")
